spark-connector/spark-common/build.gradle.kts

```diff
@@ -39,6 +39,9 @@ val scalaCollectionCompatVersion: String = libs.versions.scala.collection.compat
 
 dependencies {
   implementation(project(":catalogs:catalog-common"))
+  implementation(project(":catalogs:catalog-jdbc-common")) {
```

Comment:
`catalogs:catalog-jdbc-common` is the module of Gravitino server, why adding this dependences?

Reply:
could you remove this?

Reply:
remove this will cause integration tests to fail. https://github.com/apache/gravitino/actions/runs/12900772663/job/35971803812?pr=6212

Reply:
we need to fix the IT failures, we could checkout the errors in server side.